__tests__/app.test.js

```diff
@@ -176,39 +175,43 @@ describe("GET /api/articles/:article_id/comments", () => {
       .expect(400)
       .then(({ res }) => expect(res.statusMessage).toBe("Bad Request"));
   });
+  it("responds with a 404 and a message when article is not found", () => {
+    return request(app)
+      .get("/api/articles/4/comments")
+      .expect(404)
+      .then(({ text }) => {
+        expect(JSON.parse(text)).toEqual({ message: "Article Not Found or No Comments Exist" });
```

Comment:
Ideally, this should display an empty array - your frontend will handle the 'error message'. As far as we're concerned, this is really a 200 - because it's not an error to not have comments